FueledUtils/SwiftUI/EdgeInsets+Helpers.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2020, Fueled Digital Media, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+extension EdgeInsets {
+	public static var zero: EdgeInsets {
+		EdgeInsets(top: 0.0, leading: 0.0, bottom: 0.0, trailing: 0.0)
```

Comment:
I suppose you can use just `EdgeInsets()` instead.

Reply:
Ah, indeed - I didn't use it initially as `SwiftUI` as usually default values that are non-zero for their default initializers (like for `VStack`/`HStack`)